example-apps/my-nest-project/src/users/user-history.entity.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  History,
+  HistoryActionKind,
+  HistoryColumn,
+} from '@kittgen/nestjs-typeorm-history';
+import { Column, Entity } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class UserHistory extends User implements History {
+  @Column()
```

Comment:
Probably also good to offer to store the instance within a json blob. so that you avoid migrations for history tables.

This probably should be optional

Reply:
See comment further above